src/main/java/coffee/synyx/frontpage/plugin/islieb/ImageModel.java

```diff
@@ -0,0 +1,16 @@
+package coffee.synyx.frontpage.plugin.islieb;
+
+import org.jsoup.nodes.Element;
+
+class ImageModel {
```

Comment:
modify this class to provide a DTO class to do not expose internals (classes, etc) to the view layer
* final class
* Constructor that gets a 'String src' and the getSrc returns this src, so we do not "expose" a none basic/primitive type to the view layer